src/graph.ts

```diff
@@ -564,8 +641,7 @@ export class FactoryGraph {
      */
     createSplitContainer(item: Item, split: number): ContainerNode {
         const containers = this.getContainers(item)
-        const name = item.name + " C" + containers.size
-        const container = new ContainerNode(name, item, this, split)
+        const container = new ContainerNode("C" + containers.size, item, this, split)
```

Comment:
```suggestion
        const container = new ContainerNode(`C${containers.size}`, item, this, split)
```

Reply:
Fixed in https://github.com/lgfrbcsgo/du-factory-generator/commit/e3612727cf42e8f92e923180d0c72033bd0a1513